docs/manual/troubleshooting.tex

```diff
@@ -873,6 +873,11 @@
 
   \end{itemize}
 
+\item To quickly rebuild the Checker Framework for local development,
```

Comment:
Depending on what is being developed, this task may or may not be artifacts required. For example, it doesn't publish to Maven local.  I think you want to speed up testing use the javac wrapper script.  In that case, `copyJarsToDist` is the task that builds only what is needed for the javac script to be updated.  The name of that task doesn't convey this, so maybe we should add an additional task that only calls `copyJarsToDist` that has a better name. Maybe `buildJavacWrapperJars`?